juju/model.py

```diff
@@ -3,12 +3,14 @@
 from abc import ABC, abstractmethod
 from collections.abc import Mapping, MutableMapping
 from subprocess import run, PIPE, CalledProcessError
+from enum import Enum, unique
 
 
 class Model:
     def __init__(self, local_unit_name, relation_names, backend):
         self._cache = ModelCache()
         self._backend = backend
+        self._local_unit_name = local_unit_name
```

Comment:
It's not worth having an attribute that looks like
```
        self._local_unit_name
```
when its single job is replacing the line
```
        self.unit.name
```